tests/fast-integration/skr-svcat-migration-test/test-helpers.js

```diff
@@ -63,6 +69,85 @@ async function installBTPServiceOperatorMigrationHelmChart() {
     }
 }
 
+async function getFunctionPod(functionName) {
+    let labelSelector = `serverless.kyma-project.io/function-name=${functionName},serverless.kyma-project.io/resource=deployment`;
+    let res = {};
+    for (let i = 0; i < 30; i++) {
+        res = await listPods(labelSelector);
+        if (res.body.items.length == 1) {
+            let pod = res.body.items[0];
+            if (pod.status.phase == "Running") {
+                return pod
+            }
+        }
+        sleep(10000);
+    }
+    if (res.body.items.length != 1) {
```

Comment:
Maybe better `res.body.items.length >= 1`? If `res.body.items.length < 1` (or === 0) you would like to know, wouldn't you?

Reply:
yes, right now this block of code is structured that for a particular `Function`, I need to wait for exactly 1 `Pod` matching the `labelSelector` to be running to ensure I am not looking at some old `Pod`.

When `res.body.items.length == 0`, following error will be thrown (and step in the test calling this will fail) for example:
```
`Failed to find function svcat-auditlog-api-1 pod in 5 minutes. Expected 1 serverless.kyma-project.io/function-name=svcat-auditlog-api-1 ,serverless.kyma-project.io/resource=deployment pod with phase "Runninng" but found 0, , `
```
https://github.com/wozniakjan/kyma/blob/defabad057213abeff7f3d0bb1d7ce5712e0bba4/tests/fast-integration/skr-svcat-migration-test/test-helpers.js#L88